sphinx_gallery/tests/reference_parse.txt

```diff
@@ -1,33 +1,39 @@
 [('text',
   '\nThe Header Docstring\n====================\n\nWhen writting latex in a Python string keep in mind to escape the backslashes\nor use a raw docstring\n\n.. math:: \\sin (x)\n\nClosing this string quotes on same line',
   1),
- ('text', 'Direct first comment\nwith second line\n', 15),
- ('code', '\nimport numpy as np\n\n', 17),
- ('code', 'A = 1\n\nimport matplotlib.pyplot as plt\n\n', 21),
- ('text',
+  ('text',  'Direct first comment\nwith second line\n',  15),
```

Comment:
These look PEP8 incompatible (style regressions)

Reply:
(would be better to change only what you must to get tests to pass, helps with `blame`)

Reply:
```suggestion
  ('text', 'Direct first comment\nwith second line\n',  15),
```

Reply:
I think I fixed it. Apologies - there was one or two typos I couldn't resist fixing.